### ISSUE TITLE
Ciao-cli: create a new cluster

### ISSUE THREAD

**Issue by kaccardi** [01org/ciao#714]
Allow the user to create a new k8s cluster via the ciao-cli interface.

**Comment by kaccardi** [comment]
Depends on #709

**Comment by kaccardi** [comment]
add bat tests to test this feature end to end.

**Comment by kaccardi** [comment]
we changed our k8s strategy, so this is no longer relevant.
